examples/cpma-config.example.yaml

```diff
@@ -1,6 +1,6 @@
 ---
-Source:
-  UserName: "root"
-  HostName: "master-0.gildub1.lab.pnq2.cee.redhat.com"
-  SSHKey: "/home/gildub/.ssh/quicklab"
-outputPath: "./data"
\ No newline at end of file
+Source: "master-0.gildub1.lab.pnq2.cee.redhat.com" # can also point to directory
```

Comment:
After the first run you can make Source equal to OutputDir and CPMA will be working with the local configuration files copy.